oci/layer/generate.go

```diff
@@ -137,6 +148,10 @@ func GenerateInsertLayer(root string, target string, opaque bool, opt *MapOption
 			}
 
 			pathInTar := path.Join(target, curPath[len(root):])
+			if packOptions.TranslateOverlayWhiteouts && isOverlayWhiteout(info) {
+				return tg.AddWhiteout(pathInTar)
```

Comment:
I feel like this case deserves a log message -- yeah, it makes sense to do the conversion but I can imagine someone might be confused by the conversion happening during `umoci insert`.